dvc/repo/diff.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import unicode_literals
+
+import os
+
+
+from dvc.scm.base import FileNotInCommitError
+
+
+def _extract_dir(self, output):
+    """Extract the content of dvc tree file
+    Args:
+        self(object) - Repo class instance
+        output(object) - OutputLOCAL class instance
+    Returns:
+        dict - dictionary with keys - paths to file in .dvc/cache
+                               values -checksums for that files
+    """
+    lst = output.dir_cache
+    return {i["relpath"]: i["md5"] for i in lst}
+
+
+def _get_tree_changes(self, a_entries, b_entries):
+    result = {"del": 0, "ident": 0, "changes": 0, "new": 0, "moves": 0}
```

Comment:
If we are going with a dict, how about we at least move these  "del", "ident", etc into macros that we could use everywhere else? E.g.
```
result = {
    DIFF_DEL: 0,
    DIFF_IDENT: 0,
    ...
}
...
result[DIFF_IDENT] += 1
...
```

Reply:
good point